R/optimise.R

```diff
@@ -0,0 +1,44 @@
+library(dplyr, warn.conflicts = FALSE)
+
+utils::globalVariables(c("zone", "intervention", "total_costs", "total_cases_averted", "value", "i", "j"))
+
+#' @import dplyr
```

Comment:
This is a short-cut to namespacing every dplyr function (but can do this if preferable)